stable/acs-engine-autoscaler/Chart.yaml

```diff
@@ -2,11 +2,11 @@ apiVersion: v1
 description: Scales worker nodes within agent pools
 icon: https://github.com/kubernetes/kubernetes/blob/master/logo/logo.png
 name: acs-engine-autoscaler
-version: 1.0.0
+version: 2.0.0
```

Comment:
Does this really need to be a major version change? Doesn't seem to be breaking.

Reply:
IMO we are changing the chart's API in a breaking way by changing the names of defined template functions.

Example: 
subchart1 defines `fullname`
parent, which depends on subchart1, uses the `fullname` template function (it's global) in a configmap (or other resource) like `{{ template "fullname" . }}`

subchart1, in a minor version release, has the global defined template names changed from "fullname" to "subchart1.fullname". 

author of parent updates dependency, tries to deploy, and parent's configmap won't render because the name of the template function changed.